### PR TITLE
added keyhud plugin

### PR DESCRIPTION
**Pull request by Mandarancio**
added new plugin: [keyhud](https://codeberg.org/Mandarancio/keyhud)

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -604,6 +604,14 @@
       "type": "library",
       "version": "22.0.1"
     },
+    {
+      "id": "keyhud",
+      "description": "Simple *key HUD* for lite-xl"
```
Comment (Guldoman) [inline manifest.json:609]: Missing comma.
Same thing in the manifest in your repo.

Also, might be better to use `Lite XL` instead of `lite-xl`.

Comment (Mandarancio) [reply manifest.json]: Thank you very much, I did not pay attention when I created the manifest on my repo and then copy pasted it... sorry

I changed `lite-xl` to `Lite XL` :)

File: manifest.json [hunk 2]
```diff
@@ -604,6 +604,14 @@
       "type": "library",
       "version": "22.0.1"
     },
+    {
+      "id": "keyhud",
+      "description": "Simple *key HUD* for Lite XL",
+      "mod_version": "3",
+      "remote": "https://codeberg.org/Mandarancio/keyhud:984f61c56e409ad62b92dff6fa23f399f761d9ab",
```
Comment (Guldoman) [inline manifest.json:611]: You also need to update the commit hash, or `lpm` will try to use the old version with the missing comma:
```suggestion
      "remote": "https://codeberg.org/Mandarancio/keyhud:1560d11fb5485aea264ea742e7ee6cd7693f03e0",
```

**Comment by Guldoman** [comment]
Looks awesome!

Only suggestion is that for configurable things, you should use `config.plugins.keyhud` with `common.merge` like how most plugins do it.

**Pull request merged**